js/main.js

```diff
@@ -1,2 +1,29 @@
 /* global data */
 /* exported data */
+var $form = document.querySelector('form');
+$form.addEventListener('submit', submitButton);
+
+var $photo = document.querySelector('img');
+var $photoURL = document.querySelector('#photoURL');
+$photoURL.addEventListener('input', updateURL);
+
+function updateURL(event) {
+  var src = $photoURL.value;
+  $photo.setAttribute('src', src);
+}
+
+function submitButton(event) {
+  event.preventDefault();
+  data.nextEntryId += 1;
```

Comment:
You should do this after you use the value of the property. With it like this, your entries probably start at 2, even though this starts at 1